docs/manual/scala/guide/cluster/DomainModelling.md

```diff
@@ -1,19 +1,121 @@
 # Domain Modelling
 
+We recommend reading [[Event Sourcing and CQRS|ES_CQRS]] as a prerequisite to this section.
+
+In this section we will guide you through all the necessary steps to model an [Aggregate](https://martinfowler.com/bliki/DDD_Aggregate.html), as defined in Domain-Driven Design, using Akka Persistence Typed and following the CQRS principles.
+
+Lagom embraces Event Sourcing and CQRS by promoting the usage of Akka Persistence Typed in combination the CQRS principle. Akka Persistence Typed provides an API for building event sourced actors, but not necessarily for CQRS Aggregates. To build CQRS applications we need to apply a few rules to our design.
+
+We will explain the API and concepts while building a simplified shopping cart model. A full-fledge shopping cart sample can be found on our [samples repository](https://github.com/lagom/lagom-samples/tree/1.6.x/shopping-cart/shopping-cart-scala).
+
+You will learn, how to:
+
+* model a CQRS Aggregate using Akka Persistence Typed
```

Comment:
Not sure if you need to list all the items here or if it suffices to say something in the line of "you will learn how to create a persistence write-side using Akka Persistence Typed that is compatible with Lagom". The reason I'm in doubt about it is that it looks like this list is about things that should belong in Akka's documentation (for example "lookup instances on the cluster and interact with them").

Reply:
I will review that section. 

On the other hand, we are replacing the `PersistenceEntityRegistry` by `ClusterSharding`. So we need to talk about `ClusterSharding`. We need to register the entity and use it to lookup. 

Akka also mentions it, but it's not the only possible usage of Persistence. In Lagom, we want users to use it with `ClusterSharding`. That's one of the added values of Lagom. It makes clustering easier on dev-mode. 

Cluster sharding is also important to protect against data corruption in a multi-node setup.